File: app/src/main/java/com/adenilson/testeandroid/networking/webservices/investiment/ScreenResponse.java
package com.adenilson.testeandroid.networking.webservices.investiment;

import java.util.List;

/**
 * @author: Adenilson N. da Silva Junior
 * On date: 09/06/2018
 */

public class ScreenResponse {

    private String title;
    private String fundName;
    private String whatIs;
    private String definition;
    private String riskTitle;
    private int risk;
    private String infoTitle;
    private MoreInfoResponse moreInfo;
    private List<InfoResponse> info;
    private List<InfoResponse> downInfo;

    public String getTitle() {
        return title;
    }

    public String getFundName() {
        return fundName;
    }

    public String getWhatIs() {
        return whatIs;
    }

    public String getDefinition() {
        return definition;
    }

    public String getRiskTitle() {
        return riskTitle;
    }

    public int getRisk() {
        return risk;
    }

    public String getInfoTitle() {
        return infoTitle;
    }

    public MoreInfoResponse getMoreInfo() {
        return moreInfo;
    }

    public List<InfoResponse> getInfo() {
        return info;
    }

    public List<InfoResponse> getDownInfo() {
        return downInfo;
    }
}

File: SantanderInvestimento/app/src/main/java/br/com/santander/santanderinvestimento/investiment/data/entity/ScreenResponse.kt
package br.com.santander.santanderinvestimento.investiment.data.entity

import br.com.santander.santanderinvestimento.investiment.domain.entity.Info
import br.com.santander.santanderinvestimento.util.JSONConvertable
import com.google.gson.annotations.SerializedName

class ScreenResponse(@SerializedName("title") val title: String,
                     @SerializedName("fundName") val fundName: String,
                     @SerializedName("whatIs") val whatIs: String,
                     @SerializedName("definition") val definition: String,
                     @SerializedName("riskTitle") val riskTitle: String,
                     @SerializedName("risk") val risk: Int,
                     @SerializedName("infoTitle") val infoTitle: String,
                     @SerializedName("info") val info: List<Info>,
                     @SerializedName("downInfo") val downInfo: List<Info>) : JSONConvertable